### ISSUE TITLE
List of changesets flickers when creating a campaign

### ISSUE THREAD

**Issue by mrnugget** [sourcegraph/sourcegraph#8133]
(I know that we talked about this before and I know that @christinaforney mentioned this but I couldn't find an issue for it. Feel free to close it if it's a duplicate.)

When creating a campaign the list of changesets flickers while they're being published. See GIF here:

![flicker_campaign_creation](https://user-images.githubusercontent.com/1185253/73438028-a8b1f880-434d-11ea-9b76-07ebdadff75c.gif)

That not only looks bad, but it also makes it hard to follow what's happening.

(In addition to that: I think it would be great if the tab switches to "Changesets" when hitting the "Create" button. That way the user can see that the changesets are now being "published". Let me know if I should create a separate ticket for this.)

**Comment by felixfbecker** [comment]
Yeah, I noticed this too. We need to change `FilteredConnction` to not show the loader when refreshing (data already available), but keep the stale data.
